### PR TITLE
Bug Fixed rax_dns_record 

### PR DESCRIPTION
**Pull request by shitalmule04**
##### SUMMARY

All bug is fixed present in rax_dns_record file

##### ISSUE TYPE

 - Bugfix Pull Request

##### COMPONENT NAME

rackspace/rax_dns_record
##### ANSIBLE VERSION

```
2.5
```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

File: lib/ansible/cli/config.py [hunk 1]
```diff
@@ -1,6 +1,24 @@
+# (c) 2017, Ansible by Red Hat, Inc.
+#
+# Ansible is free software: you can redistribute it and/or modify
```
Comment (dagwieers) [inline lib/ansible/cli/config.py]: We now have a one-line GPL license statement.

File: lib/ansible/modules/cloud/amazon/aws_kms.py [hunk 2]
```diff
@@ -230,16 +234,17 @@ def assert_policy_shape(policy):
...
     argument_spec.update(dict(
-        mode = dict(choices=['grant', 'deny'], default='grant'),
-        key_alias = dict(required=False, type='str'),
-        key_arn = dict(required=False, type='str'),
-        role_name = dict(required=False, type='str'),
-        role_arn = dict(required=False, type='str'),
-        grant_types = dict(required=False, type='list'),
-        clean_invalid_entries = dict(type='bool', default=True),
+        mode=dict(choices=['grant', 'deny'], default='grant'),
+        key_alias=dict(required=False, type='str'),
```
Comment (dagwieers) [inline lib/ansible/modules/cloud/amazon/aws_kms.py]: You don't need `required=False`, it's the default. And we tend to start with the parameter type.

**Review by dagwieers — CHANGES_REQUESTED** [review]
LGTM

**Comment by gundalow** [comment]
As agreed in 2017-12-07 Core meeting, we've done bulk pep8 fixes across all modules in `devel`.

This was done in https://github.com/ansible/ansible/pull/33682 and https://github.com/ansible/ansible/pull/33690 which have been merged into `devel`. because of this you will need to rebase your PR. Thanks in advance.

File: lib/ansible/modules/cloud/rackspace/rax_dns_record.py [hunk 3]
```diff
@@ -127,6 +127,7 @@
                                       rax_find_server,
                                       rax_required_together,
                                       rax_to_dict,
+                                      setup_rax_module,)
```
Comment (gundalow) [inline lib/ansible/modules/cloud/rackspace/rax_dns_record.py:130]: Is this right?

**Pull request closed without merging**